.github/workflows/flow-sanitizer.yml

```diff
@@ -15,6 +15,6 @@ jobs:
     with:
       container: redisfab/clang:16-x64-bullseye
       test-config: ${{ inputs.flow-config }}
-      get-redis: ''
+      get-redis: 'skip getting redis'
```

Comment:
Why changing?

Reply:
Problems with manually triggering some flows. Empty is useful so I changed the skip string